connectors/migrations/20240102_github_add_issues_discussions_parents.ts

```diff
@@ -38,7 +38,7 @@ async function updateParents(connector: ConnectorModel) {
   for (const chunk of discussionChunks) {
     await Promise.all(
       chunk.map(async (d) => {
-        const documentId = getDiscussionDocumentId(
+        const documentId = getDiscussionInternalId(
```

Comment:
I reckon it's the same but it's a bit dangerous since we really really really don't want to change the documentId

Reply:
i'm confident in this but can rollback

Reply:
on second thought I rollbacked